packages/react-menu/src/MenuGroup/MenuGroup.types.ts

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react';
+import { ComponentProps } from '@fluentui/react-utils';
```

Comment:
suggestion: I'd re-create this type within the package with omission of the GenericProps which destroys type checking